### ISSUE TITLE
CombineLayer mismatches feature dims

### ISSUE THREAD

**Issue by vieting** [rwth-i6/returnn#719]
I have a case where the inputs to the `CombineLayer` are of shape `(B, F, T)` and `(1, F)`. In this case, it does currently not work to find the matching dims as the `1` axis is mapped to `F` of input 1 and then, the `F` axis cannot be mapped anymore.

**Comment by albertz** [comment]
> My issue was resolved with [rwth-i6/pytorch-to-returnn#58](https://github.com/rwth-i6/pytorch-to-returnn/pull/58). Should we close the issue or do you want to keep it open since it should still work with RETURNN?

No, this problem is not solved on RETURNN. You just avoid the RETURNN problem (which still exists) in pytorch-to-returnn now, so your linked issue is not relevant here anymore.
